inc/enqueue.php

```diff
@@ -62,8 +62,8 @@ function uds_wp_admin_scripts() {
 		$fa_js_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/fontawesome/all.min.js' );
 		wp_enqueue_script( 'uds-wordpress-fa-scripts', get_template_directory_uri() . '/js/fontawesome/all.min.js', array(), $fa_js_version, true );
 
-		$js_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/core-list-block.js' );
-		wp_enqueue_script( 'uds-wordpress-admin-scripts', get_template_directory_uri() . '/js/core-list-block.js', array(), $js_version );
+		$js_core_list_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/core-list-block.js' );
```

Comment:
We should be careful about working in changes that are not related to the focus of the branch...presuming this is not related to foldable cards.